security/index.adoc

```diff
@@ -0,0 +1,85 @@
+[[security-intro]]
+= Introduction
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:sourcedir: ../
+
+toc::[]
+
+[[security-intro-about-this-guide]]
+== About This Guide
+
+This guide provides a high-level walkthrough of the container security measures
+available in {product-title}, from the host layer to the container and
+orchestration layer to the user build and application layer. After reading this
```

Comment:
The above sentence confused me a little. I think it's the list... Perhaps:

"This guide provides a high-level walkthrough of the container security measures available in {product-title}, including solutions for the host layer, the container and orchestration layer, and the build and application layer."